src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -121,32 +134,64 @@ class BaseCreateCourseForm extends React.Component {
             }
             required
           />
-          <Field
-            name="enrollmentTrack"
-            component={RenderSelectField}
-            options={this.getEnrollmentTrackOptions()}
-            label={
-              <FieldLabel
-                id="enrollment-track-label"
-                text="Enrollment track"
+          {usingCourseType ? (
+            <React.Fragment>
+              <Field
+                name="type"
+                component={RenderSelectField}
+                options={courseTypeOptions}
+                label={
+                  <FieldLabel
+                    id="course-type-label"
+                    text="Course Type"
```

Comment:
I know to us engineers, this is now a "Course Type" but to the user, nothing has changed, right? Might want to keep labelling it as "Enrollment track"?  (also those two strings have different capitalization schemes)

I guess you have to differentiate the course from the run. But I don't know what language users would be comfortable here. "Course type" is not a thing I think they've seen before. I guess I'm saying this might be a good question for Product.

Reply:
I can add in a TODO here as well to mark it as a standing product question. Similar to help text, I'm not sure what the best language is yet.